hq/app/model/Serializers.scala

```diff
@@ -0,0 +1,30 @@
+package model
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Format, JsPath, Json, Reads}
+
+object Serializers {
+
+  implicit val snykErrorFormat: Reads[SnykError] = Json.reads[SnykError]
+
+  implicit val snykOrgFormat: Reads[SnykOrganisation] = Json.reads[SnykOrganisation]
+
+  implicit val snykProjectFormat: Reads[SnykProject] = (
```

Comment:
Are you sure the macro won't work here as well? Options in the target case class denote optional fields, it'll be `Some` if present, `None` if absent.

Reply:
They aren't returned as part of the json response (they had to be specified in the request).  They will definitively always be None.  The values are then 'enriched' on to the returned object after parsing.